net/portmapper/upnp_test.go

```diff
@@ -50,7 +51,7 @@ func TestParseUPnPDiscoResponse(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := parseUPnPDiscoResponse([]byte(tt.headers))
+			got, err := parseUPnPDiscoResponse([]byte(tt.headers), logger.Discard)
```

Comment:
as a bonus, these tests would then actually start testing things, rather than throwing it away